python/src/aiconfig/editor/client/src/components/prompt/prompt_input/attachments/AttachmentUploader.tsx

```diff
@@ -15,15 +15,65 @@ type Props = {
   onCancel?: () => void;
 };
 
-async function uploadFile(_file: File) {
-  // TODO: Implement
-  return {
-    url: "https://s3.amazonaws.com/files.uploads.lastmileai.com/uploads/cldxsqbel0000qs8owp8mkd0z/2023_12_1_21_23_24/942/Screenshot 2023-11-28 at 11.11.25 AM.png",
-  };
+// s3 file uris cannot have '+' character, so replace with '_'
+function sanitizeFileName(name: string) {
+  return name.replace(/[_+]/g, "_");
+}
+
+export function getTodayDateString(): string {
+  const date = new Date();
+  const dateString = `${date.getFullYear()}_${
+    date.getMonth() + 1
+  }_${date.getDate()}`;
+  const timeString = `${date.getUTCHours()}_${date.getUTCMinutes()}_${date.getUTCSeconds()}`;
```

Comment:
Do you feel like adding the timeString for seconds is overkill and we can just use the date instead? Not a blocker

Reply:
Eh, I think it's fine to be extra safe. We could also look at adding some additional 'unique' identifier based on session or something later